extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -145,7 +146,10 @@ protected function getSourceForChildren() {
 	protected function getValueWithDefault( \SimpleXMLElement $xmlNode ) {
 		$value = $this->extractDataFromSource( $xmlNode );
 		if ( !$value && $xmlNode->{self::DEFAULT_TAG_NAME} ) {
```

Comment:
Maybe we could switch places for those two ifs, to make return statement less confusing